pkg/azurefile/nodeserver.go

```diff
@@ -435,6 +504,12 @@ func (d *Driver) NodeUnstageVolume(_ context.Context, req *csi.NodeUnstageVolume
 			return nil, status.Errorf(codes.Internal, "failed to unmount staging target %s: %v", targetPath, err)
 		}
 	}
+
+	klog.V(2).Infof("NodeUnstageVolume:remove direct volume mount info %s from %s", volumeID, stagingTargetPath)
```

Comment:
I could only see there is kata-cc mount in NodePublishVolume, then why there is `directvolume.Remove` in NodeUnstageVolume

Reply:
In NodeStageVolume, the mountinfo is saved on a /globalmount path. Which is removed in NodeUnstageVolume.